openwisp_notifications/handlers.py

```diff
@@ -29,21 +29,23 @@ def notify_handler(verb, **kwargs):
 
     target_org = getattr(kwargs.get('target', None), 'organization_id', None)
 
-    where = Q()
+    where = Q(is_superuser=True)
+    where_group = Q()
     if target_org:
-        where = Q(is_staff=True) & Q(openwisp_users_organization=target_org)
+        where = where | (Q(is_staff=True) & Q(openwisp_users_organization=target_org))
+        where_group = Q(openwisp_users_organization=target_org)
```

Comment:
I implemented it for the following use case. 
Suppose a person from administration or management needs to be notified for network events(this can be further customised with new features) just for sake of redundancy. While he/she does not necessarily need staff privilege to login and manage the network due to his/her unfamiliarity with the concept, but he/she can receive notifications.  Therefore a user can be created for such personnel which can be added to a group. Therefore they can receive notifications even if they are not able to login to the admin dashboard. 

If there is no possibility this can happen, I will amend it.

Reply:
excellent this is good. I mean a user might demand that his device should be created and want to also be able to receive notifications when the device is up and/or down without logging in. this is good. :+1:

Reply:
at the moment only staff users will receive notifications but we may send notifications to non staff users as well in the near future.

Reply:
Should I change this?